Sources/Tabs/TabManagerCoordinator.swift

```diff
@@ -64,7 +64,6 @@ final class TabManagerCoordinator: NSObject,
         guard walletAvailabilityService.shouldShowFeatureOnUniversalLink else {
             return
         }
-        addWalletTab()
```

Comment:
We not replacing this with the logic we discussed earlier?

Reply:
I rebased before pushing my changes and it seems it undid that.. I've amended it now, sorry

Reply:
Easily done 🙂